control-plane/tests/test_cli_inspection.py

```
@@ -1644,62 +1562,22 @@ def post_json(path: str, payload: dict[str, object]) -> dict[str, object]:
     def test_long_running_runtime_surface_exposes_cited_advisory_review_routes(self) -> None:
-        store, _ = make_store()
-        service = AegisOpsControlPlaneService(
-            RuntimeConfig(
-                host="127.0.0.1",
-                port=0,
-                postgres_dsn="postgresql://control-plane.local/aegisops",
-                wazuh_ingest_shared_secret="reviewed-shared-secret",  # noqa: S106 - test fixture secret
-                wazuh_ingest_reverse_proxy_secret="reviewed-proxy-secret",  # noqa: S106 - test fixture secret
-            ),
-            store=store,
+        store, service, promoted_case, evidence_id, _ = self._build_phase19_in_scope_case(
```

Ruff (0.15.9)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)